laws.md

```diff
@@ -70,3 +70,5 @@ New Judges are not bound by the decisions of old Judges. New Judges may, however
 It is important, for parsing, that no other non-space characters be included in the comment body. The comment must be attached to the PR itself, and not associated with any specific file.
 
 **305.** No player may be a judge in a dispute in which they are involved.
+
+**306.** PARSING VOTES: Only the most recent vote will be counted. All other votes by a given player will be ignored.
```

Comment:
This should be 307 as there is an active proposal for 306. Even if 306 is declined, this would still be 307 as I understand.